src/components/DeleteSessionDialog.vue

```diff
@@ -0,0 +1,50 @@
+<script setup>
+import { fetchApiCall } from '@/utils/api';
+const props = defineProps([ 'modelValue', 'deleteId'])
+const emit = defineEmits(['update:modelValue', 'reloadSession'])
+
+function closeDialog() { emit('update:modelValue', false)}
+
+async function confirmDeleteSession() {
+  if (props.deleteId != -1) {
+    const url = 'http://127.0.0.1:8000/api/datasessions/' + props.deleteId
```

Comment:
I think I mentioned this in Carolina's PR, but we probably want to make a story to get these URLs into a central place (like the store) and read in from environment variables